src/plugins/pishock.rs

```diff
@@ -102,11 +103,12 @@ async fn send_shock(
     }
 }
 
-async fn send_shocks(
+async fn send_commands(
     config: &Arc<ConfigHandle<CoreConfig>>,
     intensity: f32,
     duration: u8,
     activity_tx: &mpsc::Sender<u8>,
+    op: u8,
```

Comment:
Instead of taking a u8 where people would have to pull up documentation, it'd be better to accept an enum, something like this:

```rust
#[derive(Debug)}
enum Operation {
    Shock,
    Vibrate,
}

impl Operation {
    fn into_byte(self) -> u8 {
        match self {
            Operation::Shock => 0,
            Operation::Vibrate => 1,
        }
    }
}
```